index.html

```diff
@@ -8,17 +8,18 @@
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
-  <h1>Awesome Books</h1>
+  <h1> All awesome Books</h1>
   <div class="awesomeList"></div>
   <div class="InputForm">
+    <h2>Add a new book.</h2>
     <input type="text" id="title" placeholder="Title"/>
     <br>
     <input type="text" id ="author"  placeholder="Author"/>
     <br>
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you add some validations on your `inputs` for preventing users to submit empty forms.